cms/static/cms/sass/components/pagetree/_tree.scss

```diff
@@ -823,20 +875,27 @@
         // we need to add one pixel in to negate the negative margin
         // value to get the borders to work visually
         height: $pagetree-cell-height + 1;
-        padding: $pagetree-anchor-padding;
-        margin-top: -1px;
-        border: 1px solid $gray-lighter;
-        border-right: none;
+        padding-block: $pagetree-anchor-padding-block;
+        padding-inline: $pagetree-anchor-padding-inline;
+        margin-block-start: -1px;
+        border-block: 1px solid $gray-lighter;
+        border-inline: 1px solid $gray-lighter;
+        border-inline-end: none;
         background-color: $white;
         background-image: url('../../img/pagetree/tree-li-drag.gif');
         background-repeat: no-repeat;
-        background-position: -1px center;;
+        background-position: left center;;
```

Comment:
```suggestion
        background-position: inline-start center;
```

Would this move the drag handle to the correct position in RTL mode?

Reply:
It does not work. Maybe something like this is needed:
```
// RTL correction
html[dir="rtl"] .jstree-django-cms .jstree-anchor {
    background-position: left center;
}
```

Reply:
Yeah, logical properties are not supported in everything:
```
        [dir=rtl] & {
            // no direct logical property for background-position
            background-position: right center;
        }
```
Some don't even exist, some are not widely supported.

I'm not sure why RTL is not working. I tried it on a couple of browsers and it looks fine. I tried it with/without django-admin-style and it's still working.

My Firefox version:  Mozilla Firefox 123.0
My node version: 18.19.0

I generated CSS files and bundled JS with gulp:
```
    "gulp": "^4.0.2",
        "gulp-clean-css": "^3.0.4",
        "gulp-eslint": "^3.0.1",
        "gulp-iconfont": "11.0.0",
        "gulp-iconfont-css": "3.0.0",
        "gulp-if": "1.2.5",
        "gulp-plumber": "^1.1.0",
        "gulp-postcss": "^6.4.0",
        "gulp-sass": "^5.0.0",
        "gulp-sourcemaps": "2.4.1",
        "gulp-util": "3.0.5",
``` 

![Screenshot from 2024-02-29 18-37-55](https://github.com/django-cms/django-cms/assets/50373064/d26662b1-2d28-4b46-bd9a-36e2107549b1)
![Screenshot from 2024-02-29 18-38-08](https://github.com/django-cms/django-cms/assets/50373064/35e49f66-801c-4683-afc3-2011c9f95626)

May I know how to reproduce your results?